1/1.12/Dockerfile

```diff
@@ -1,9 +1,44 @@
-FROM gcr.io/google-appengine/debian8
+FROM gcr.io/google-appengine/debian9
 
 ENV NGINX_VERSION 1.12.2*
 
-RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62 \
-	&& echo "deb http://nginx.org/packages/debian/ jessie nginx" >> /etc/apt/sources.list \
+# grab gosu for easy step-down from root
+ENV GOSU_VERSION 1.10
```

Comment:
Do we need/use gosu in this repo?

Reply:
No, we don't. I will remove it in the next version.